content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -135,7 +144,7 @@ The real-time validation tool allows submitters to check their filled-in metadat
 
 [![](../_static/images/howto_guides/portal_guide/find-replace.png)](../_static/images/howto_guides/portal_guide/find-replace.png)
```

Comment:
I guess this screenshot (and probably some others, like the import/export one) needs to be updated with the UI reorganization that came with the metadata suggester. Not saying it needs to be done with these changes necessarily, but I just wanted to mention it.

Reply:
Yes.. I thought I had an issue... but apparently not! Making one now!